plugin.rb

```diff
@@ -63,14 +63,16 @@ def top_posters
       data-bsa-placement="sitepointforums"></script>
     EOS
 
-  sitepoint_site_customization = SiteCustomization.find_or_create_by({
-    name: "SitePoint Crawler links",
-    header: header,
-    mobile_header: header,
-    enabled: true,
-    user_id: User.first.id,
-    head_tag: head_tag
-  })
+	unless User.nil?
+	sitepoint_site_customization = SiteCustomization.find_or_create_by({
+		name: "SitePoint Crawler links",
+		header: header,
+		mobile_header: header,
+		enabled: true,
+		user_id: User.first.id,
+		head_tag: head_tag
+	})
+	end
   # cleanup old customizations
   SiteCustomization.where.not(id: sitepoint_site_customization.id).delete_all
```

Comment:
This should be part of the above if block. `sitepoint_site_customization` will not be there is the conditional above fails